code/WorkInProgress/Cael_Aislinn/Rust/circuits_and_design.dm

```diff
@@ -124,3 +124,20 @@ datum/design/rust_injector
 	materials = list("$glass" = 2000, "sacid" = 20, "$plasma" = 3000, "$uranium" = 2000)
 	category = "Misc"
 	build_path = "/obj/item/weapon/circuitboard/rust_core"
+
+//Gyrotron controller board.
+/obj/item/weapon/circuitboard/rust_gyrotron_control
+	name = "Circuit board (RUST gyrotron controller)"
+	build_path = "/obj/machinery/computer/rust_gyrotron_controller"
+	origin_tech = "programming=4;engineering=4"
+
+/datum/design/rust_gyrotron_control
+	name = "Circuit Design (RUST gyrotron controller)"
+	desc = "Allows for the construction of circuit boards used to build a gyrotron control console for the RUST fusion engine."
+	id = "rust_gyrotron_control"
+	req_tech = list("programming" = 4, "engineering" = 4)
+	build_type = IMPRINTER
+	materials = list("$glass" = 2000, "sacid" = 20)
```

Comment:
The other designs will have been edited on the repo already.
The only issue is that it will cause a compile failure because your head is behind.
You'll have to rebase, then change this to use the define.